UI_Engine/Query/Items.cs

```diff
@@ -107,7 +107,7 @@ public static IEnumerable<Type> ConstructableRequestItems()
         {
             return Engine.Reflection.Query.BHoMTypeList()
                 .Where(x => x != null && x.Namespace.StartsWith("BH.oM.Adapters") && !x.IsNotImplemented() && !x.IsDeprecated() && !x.IsEnum && !x.IsAbstract)
```

Comment:
Do not think we should/need to filter only items in the adapter namespace. The `FitlerRequest` for example sits in `BH.oM.Data` and there are other requests available in Structures for example.

```suggestion
                .Where(x => x != null  && !x.IsNotImplemented() && !x.IsDeprecated() && !x.IsEnum && !x.IsAbstract)
```

Reply:
Hmm, seems to work fine for me functionality wise, even though the code would say something else:

![image](https://user-images.githubusercontent.com/26874773/93205485-2c431d00-f758-11ea-94eb-7184587019a5.png)

Reply:
That is only the create methods, not the property assignment methods (automatic creates). That is at least the only ones showing up for me.

Reply:
Right. I stopped at the stage where the `Create` methods were needed, so simply added all of them. But yeah, would be great to be able to get rid of them and stick to the auto creates.